build.gradle.kts

```
@@ -1,13 +1,13 @@
 plugins {
-    val kotlinVersion = "1.4.21"
+    val kotlinVersion = "1.5.21"
     kotlin("jvm") version kotlinVersion
     kotlin("plugin.serialization") version kotlinVersion
 
-    id("net.mamoe.mirai-console") version "2.0-RC"
+    id("net.mamoe.mirai-console") version "2.6.7"
 }
 
 group = "net.mamoe"
-version = "0.5.1"
+version = "0.5.2"
```

有新功能，则应更新次版本号，即
```
version = "0.6.0"
```
不过 PR 也不应该修改版本号，因为不确定该 PR 会在哪个版本合并
```suggestion
version = "0.5.1"
```